File: src/main/java/me/nepnep/nepbot/message/command/commands/animals/BigMonkey.java
package me.nepnep.nepbot.message.command.commands.animals;

import me.nepnep.nepbot.message.command.ICommand;
import me.nepnep.nepbot.message.command.Category;
import net.dv8tion.jda.api.Permission;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

import java.util.List;

public class BigMonkey implements ICommand {
    @Override
    public void execute(List<String> args, GuildMessageReceivedEvent event) {
        event.getChannel().sendMessage("https://bigmonkey.monster/").queue();
    }
    @Override
    public String getInvoke() {
        return "bigmonkey";
    }
    @Override
    public Permission requiredPermission() {
        return null;
    }
    @Override
    public Category getCategory() {
        return Category.ANIMALS;
    }
    @Override
    public String getDescription() {
        return "A big monkey";
    }
}

File: src/main/kotlin/me/nepnep/nepbot/message/command/commands/animals/BigMonkey.kt
package me.nepnep.nepbot.message.command.commands.animals

import me.nepnep.nepbot.message.command.Category
import me.nepnep.nepbot.message.command.AbstractCommand
import net.dv8tion.jda.api.entities.GuildMessageChannel
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class BigMonkey : AbstractCommand(
    "bigmonkey",
    Category.ANIMALS,
    "A big monkey"
) {
    override fun execute(args: List<String>, event: MessageReceivedEvent, channel: GuildMessageChannel) {
        channel.sendMessage("https://bigmonkey.monster/").queue()
    }
}